notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>Maybe say that <code>OpenSet</code>  corresponds to definition 1) (Local parametrization).</p><p> </p><p>Instead of "in mathematics", maybe say: "Intuitively", or "roughly speaking" since this is not the exact mathematical definition of an open set.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>done</p>